rules/PointPolygon.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * This script conflates all points with all polygons using Generic Conflation.
+ */
+
+"use strict";
+
+exports.candidateDistanceSigma = 1.0; // 1.0 * (CE95 + Worst CE95);
+exports.description = "Matches generic points with polygons";
+exports.experimental = false;
+exports.matchThreshold = parseFloat(hoot.get("generic.point.polygon.match.threshold"));
+exports.missThreshold = parseFloat(hoot.get("generic.point.polygon.miss.threshold"));
+exports.reviewThreshold = parseFloat(hoot.get("generic.point.polygon.review.threshold"));
+exports.searchRadius = parseFloat(hoot.get("search.radius.generic.point.polygon"));
+exports.tagThreshold = parseFloat(hoot.get("generic.point.polygon.tag.threshold"));
+exports.writeDebugTags = hoot.get("writer.include.debug.tags");
+//exports.baseFeatureType = ""; // 
+//exports.geometryType = "";
```

Comment:
The whole geometry type design kind of breaks apart for this script...but after a few small workarounds its not too kludgy of and end result for now. See notes in `ScriptMatchCreator`.